template_feed/Microsoft.DotNet.Common.ItemTemplates/content/RazorPage/.template.config/template.json

```diff
@@ -0,0 +1,36 @@
+{
+  "author": "Microsoft",
+  "classifications": ["Web", "ASP.NET"],
+  "name": "Razor Page",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorPage",
+  "shortName": "page",
+  "sourceName": "Index",
+  "primaryOutputs": [ { "path": "Index.cshtml"}, { "path": "Index.cs" } ],
+  "defaultName": "Index",
+  "sources": {
+    "modifiers": [
+      {
+        "condition": "(no-pagemodel)",
```

Comment:
@seancpeters - this isn't working. I've tried some stuff and it still wants to output the `Index.cs` file.